### ISSUE TITLE
gabbi-run argparse help does not announce file handling functionality

### ISSUE THREAD

**Issue by cdent** [cdent/gabbi#190]
Should it?

/cc @FND

**Comment by FND** [comment]
I assume you're referring to the caveat mentioned in b1b0dfa54636d3ff61359a8704df630df9e051f2? I'm guessing the only reasonable way would be to extend the [`description`](https://github.com/cdent/gabbi/blob/042f891562fe266dba104c67e543f0a3e42489be/gabbi/runner.py#L171) with a note about multiple files - but yes, that'd be nice.

**Comment by cdent** [comment]
Yes, pretty much on extending the description, or adding a postscript (I _think_ that's possible).
